servers/rendering/shader_language.cpp

```diff
@@ -2852,7 +2852,7 @@ bool ShaderLanguage::_validate_function_call(BlockNode *p_block, const FunctionI
 									}
 								}
 								if (error) {
-									_set_error(vformat("Expected integer constant within %s..%s range.", min, max));
+									_set_error(vformat(RTR("Expected integer constant within [%s..%s] range."), min, max));
```

Comment:
```suggestion
									_set_error(vformat(RTR("Expected integer constant within [%d..%d] range."), min, max));
```

For integers you should use `%d` (though I'm surprised it was happy with `%s` before). `%s` is for Strings. (For floats it's `%f`, see https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_format_string.html#placeholder-types for details.)

Reply:
Thanks, fixed all cases where it happens (I hope).